<launch>

  <!--- Eddiebot Line Follower -->
  
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node">
    <param name="video_device" value="/dev/video1" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
  <node name="image_view" pkg="image_view" type="image_view" respawn="false">
    <remap from="image" to="/usb_cam/image_raw"/>
    <param name="autosize" value="true" />
  </node>
  <node pkg="eddiebot_line_follower" type="eddiebot_line_follower" name="eddiebot_line_follower">
    <param name="scale_angular" value="1"/>
    <param name="scale_linear" value="1"/>
    <param name="angular" value="0.3"/>
    <param name="linear" value="0.02"/>
    <param name="left_threshold" value="150"/>
    <param name="right_threshold" value="450"/>
    
  </node>

</launch>
